data/habitat_confirmations_priorities.csv

```diff
@@ -1,15 +1,15 @@
 ﻿reference_number,gazetted_name,alias_local_name,site,location,ef,date,time_start,crew_members,length_surveyed,hab_value,priority,upstream_habitat_length_m,species_codes,waypoint,comments
 1,Watson Creek,197974_ds,197974,ds,,2022-08-28,14:15,MW,350,medium,,,,,Medium value habitat. Low flow near culvert but as you get away (>50m) from it there's much better flow and habitat. Numerous deep pools and undercut banks. Not much cover in most areas. Abundant gravels suitable for spawning. Stream opens up after 100m and goes through cow pasture whre there's less cover and the stream becomes more coupled. 16:15
-2,Watson Creek,197974_us,197974,us,,2022-08-28,14:18,AI,600,medium,moderate,11395,"CO, RB",,Medium value habitat. Abundant gravels present for spawning. Lower 150-200m of stream is heavily impacted by cattle from Buckrow Angus farm. Beaver present in lower section. Numerous fry throughout. Some deep pools to 0.7m available for rearing and overwintering for rainbow fry and juvenile. Frequent debris in stream. Cattle accessing stream intermittently to top of site. 16:18

```

Comment:
this is really great and exactly what i was hoping for. I should have pulled this earlier but I had my head down to get a bunch of stuff done. We will need to test if you are able to merge my latest pushes of `main` into this or if you are going to need to copy it over from the csv again because I changed all the comments and a bunch of fish and stuff on this csv already